hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -111,19 +110,12 @@ public void doStateTransition() {
             AccountID sponsor = txnCtx.activePayer();
 
             CryptoCreateTransactionBody op = cryptoCreateTxn.getCryptoCreateAccount();
-            long balance = op.getInitialBalance();
             final var customizer = asCustomizer(op);
-            final var isLazyCreation =
-                    !op.getAlias().isEmpty() && op.getAlias().size() == EVM_ADDRESS_SIZE && !op.hasKey();
-            final var lazyCreationFinalizationFee = autoCreationLogic.getLazyCreationFinalizationFee();
-            final var minPayerBalanceRequired = balance + (isLazyCreation ? lazyCreationFinalizationFee : 0);
+            final var minPayerBalanceRequired = op.getInitialBalance();
             if (minPayerBalanceRequired > (long) ledger.getAccountsLedger().get(sponsor, BALANCE)) {
```

Comment:
I am not sure that we need this check anymore. It was introduced to check that the payer can cover one cryptoUpdate operation for hollow account finalisation.